deploy/charts/harvester/values.yaml

```diff
@@ -464,5 +464,5 @@ harvester-node-manager:
   image:
     imagePullPolicy: IfNotPresent
     repository: rancher/harvester-node-manager
-    tag: v0.1.2
+    tag: master-head
```

Comment:
should the tag not be `v0.1.3`?

Reply:
we will tag it in the release branch, for the master branch prefer to use the current tag to make it consistent like others.

Reply:
sorry yeah.. realized this is for master